src/components/common/RegAddEditOrgPerson.vue

```diff
@@ -102,104 +116,12 @@
               </div>
             </article>
 
-            <!-- Business or Corporation -->
+            <!-- Business or Corporation Look up-->
             <template v-if="isOrg">
-              <!-- Business or Corporation Unregistered in B.C. -->
```

Comment:
remove "business or corporation unregistered in B.C." since we do not want this section now, only "existing B.C. business" now
![image](https://github.com/bcgov/business-create-ui/assets/144159934/b8ce47b6-2648-4347-8701-06366e171036)

Reply:
Also, I see in the uxpin, "Don't make changes for Registries stadd. Make the changes for SBC staff and clients."